plugins/ledger/README.md

```diff
@@ -0,0 +1,3 @@
+# Ledger Plugin
+
+Little plugin that allows samson to post events to ledger
```

Comment:
mention that it needs LEDGER_URL etc ... maybe best to remove that from the global readme since nobody else will need that ...